src/examples/datepicker/DatepickerRange.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { DatepickerRange } from '@zendeskgarden/react-datepickers';
+import { Field, Label, Input, Message } from '@zendeskgarden/react-forms';
+import { compareAsc, addDays } from 'date-fns';
```

Comment:
Should `date-fns` be added to the codesandbox parameters?

Reply:
It looks like it's a dependency of the [Datepicker package](https://github.com/zendeskgarden/react-components/blob/main/packages/datepickers/package.json#L26). I'm not sure of this answer, @austin94 what say ye?

Reply:
It's necessary for some of the logic used in the example. I could see value including it for the codesandbox examples.